Formula/bootloadhid.rb

```diff
@@ -0,0 +1,19 @@
+class Bootloadhid < Formula
+  desc "HID-based USB bootloader for AVR microcontrollers"
+  homepage "https://www.obdev.at/products/vusb/bootloadhid.html"
+  url "https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz"
+  version "2012-12-08"
```

Comment:
This seems like a very old version, is the software still maintained?

Reply:
Objective Development appear to be still around and doing things. Due to the relative simplicity of the program I can see it not needing to be updated since its last release - I can confirm it still works perfectly fine on 10.15.4.
We over at QMK (https://github.com/qmk) make use of this tool to flash certain keyboards we support. If it breaks, we'll know, and we may fork it to keep it going, if that's acceptable to you guys. But I don't expect it to break in the foreseeable future.

Reply:
Just wanted to make sure it's still maintained. Good to know